lib/ckb/types/script.rb

```diff
@@ -12,6 +12,13 @@ def initialize(code_hash:, args:)
         @args = args
       end
 
+      # @return [Integer]
+      def calculate_capacity
+        capacity = 1 + (@args || []).map(&:bytesize).reduce(0, &:+)
+        capacity += Utils.hex_to_bin(@code_hash).bytesize if @code_hash
```

Comment:
The name is confusing and inconsistent. Capacity should always use Shannon as the uint.

Reply:
fixed